shell_scripts/unzip_corpus.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
```

Comment:
It expects the structure resulting in a direct download of our private Google Drive.
This script is only for us to use since we'll never release the data with names attached to it, and will likely never be called again after we publish the corpus.

I just included it to document our process, not for reproducibility